### PR TITLE
Add concrete content handler

### PR DESCRIPTION
**Pull request by dolfelt**
Defaults to using JSON content handler.

### CODE REVIEW

**Comment by ameech** [comment]
Seems like a logical default.

**Comment by dolfelt** [comment]
Yeah, a default is needed. `ContentHandler` is abstract, so it fails to run using it.

**Comment by elazar** [comment]
:+1:

File: src/Middleware/DefaultCollection.php [hunk 1]
```diff
@@ -15,7 +16,7 @@ public function __construct()
             ResponseSender::class,
             ExceptionHandler::class,
             RouteHandler::class,
-            ContentHandler::class,
+            JsonContentHandler::class,
```
Comment (shadowhand) [inline src/Middleware/DefaultCollection.php]: Why not both `JsonContentHandler` and `FormContentHandler`? They don't conflict.

Comment (dolfelt) [inline src/Middleware/DefaultCollection.php]: Who uses form data anymore @shadowhand!? :stuck_out_tongue_closed_eyes:

Comment (shadowhand) [inline src/Middleware/DefaultCollection.php]: Anyone who uses Spark as a HTML generator? :stuck_out_tongue_winking_eye:

Comment (dolfelt) [inline src/Middleware/DefaultCollection.php]: Touché. Way to one up my tongue emoji too.

File: src/Middleware/DefaultCollection.php [hunk 2]
```diff
@@ -3,8 +3,10 @@
 namespace Spark\Middleware;
 
 use Relay\Middleware\ResponseSender;
-use Spark\Handler\ContentHandler;
+use Spark\Handler\ActionHandler;
```
Comment (shadowhand) [inline src/Middleware/DefaultCollection.php]: Where did `ActionHandler` come from?

File: src/Middleware/DefaultCollection.php [hunk 3]
```diff
@@ -15,7 +17,8 @@ public function __construct()
             ResponseSender::class,
             ExceptionHandler::class,
             RouteHandler::class,
-            ContentHandler::class,
+            JsonContentHandler::class,
+            FormContentHandler::class,
             ActionHandler::class,
```
Comment (shadowhand) [inline src/Middleware/DefaultCollection.php]: Oh nevermind.

File: src/Middleware/DefaultCollection.php (hunk 2 above)
Comment (elazar) [inline src/Middleware/DefaultCollection.php]: It was used in the class body, the `use` statement was just missing, and this wasn't covered by tests, so it went unnoticed.

**Pull request merged**